test/features/math/simple.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var mjAPI = require("../../../lib/mj-single.js");
+var assert = require('../../utils/assert.js');
+
+var testData = [
+        {
+            query: {
+                math: 'E=mc^2',
+                mml: true,
+                speakText: true
+            },
+            response: {
+                "mml": "<math xmlns=\"http://www.w3.org/1998/Math/MathML\" display=\"block\" alttext=\"upper E equals m c squared\">\n  <semantics>\n    <mrow>\n      <mi>E</mi>\n      <mo>=</mo>\n      <mi>m</mi>\n      <msup>\n        <mi>c</mi>\n        <mn>2</mn>\n      </msup>\n    </mrow>\n    <annotation encoding=\"application/x-tex\">E=mc^2</annotation>\n  </semantics>\n</math>",
```

Comment:
> Are there automatic tests for mathjax

Of course there are -- well over 1000. https://github.com/mathjax/MathJax-test/tree/master/testsuite

> I predict that you'll enjoy automatic tests, once you get used to them.

We have been used to them for a few years :-) We would love to refactor the test suite when we have the time (yeah right...), which includes separating the pure JS tests from the Selenium tests.